Cargo.toml

```diff
@@ -30,17 +31,21 @@ slip21 = "0.2.0"
 bitcoin_hashes = "0.10.0"
 
 # Used for ContractHash::from_json_contract.
-serde_json = { version = "<=1.0.44", optional = true }
+serde_json = { version = "1.0", optional = true }
 
 serde = { version = "1.0", features=["derive"], optional = true }
 
+# This should be an optional dev-dependency (only needed for integration tests),
+# but dev-dependency cannot be optional, and without optionality older toolchain try to compile it and fails
+elementsd = { version="0.3.0", features=["0_21_0","bitcoind_22_0"], optional = true }
```

Comment:
I would like this to be dev-dependency but as written in the comment I am afraid it cannot be...

Reply:
It doesn't, there were some efforts to support it here  https://github.com/RCasatta/bitcoind/pull/31 but in the end, I thought it's fine for integration tests to run on stable

Reply:
Agreed, it's fine for the integration tests to only be on stable